src/Argu/Attributes.fs

```diff
@@ -128,6 +129,30 @@ type EqualsAssignmentAttribute () =
 [<AttributeUsage(AttributeTargets.Property, AllowMultiple = false)>]
 type ColonAssignmentAttribute () = 
     inherit CustomAssignmentAttribute(":")
+    
+/// Use a custom separator for parameter assignment.
+/// e.g. '--param<separator>arg'
+/// Parameters can also be assigned using space as separator e.g. '--param arg'
+/// Requires that the argument should have parameters of arity 1 only.
+/// Can be used to specify any assignment separator.
+[<AttributeUsage(AttributeTargets.Property, AllowMultiple = false)>]
+type CustomAssignmentOrSpacedAttribute (separator : string) = 
+    inherit Attribute ()
+    member __.Separator = separator
+
+/// Use '--param=arg' assignment syntax in CLI.
+/// Parameters can also be assigned using space as separator e.g. '--param arg'
+/// Requires that the argument should have parameters of arity 1 only.
```

Comment:
Could you please add a negative test for arity 2 scenaria? Thanks.

Reply:
Done, and added comment clarifying arity 1 vs arity 2.